### PR TITLE
GCB-120: Add up to 5 Component Charts Views.

### PR DESCRIPTION
**Pull request by ochococo**
🚚  https://macoscope.atlassian.net/browse/GCB-120

simulator screen shot 15 04 2016 15 52 20

### CODE REVIEW

File: GrandCentralBoard/Widgets/Harvest/HarvestWidget.swift [hunk 1]
```diff
@@ -38,14 +38,18 @@ final class HarvestWidget : Widget {
...
 
-        let model = AreaBarChartViewModel(barItems: items, horizontalAxisStops: 20, horizontalAxisLabelText: "people billing:", hotizontalAxisCountLabelText: "666", centerText: nil, headerText: "HARVEST BURN REPORT", subheaderText: "Monday 28.03.2016")
+        let mainChart = AreaBarChartComponentViewModel(barItems: items, horizontalAxisCountLabelText: "20", headerText: "HARVEST BURN REPORT", subheaderText: "Monday 28.03.2016")
+
+        let history = AreaBarChartComponentViewModel(barItems: items, horizontalAxisCountLabelText: "12", headerText: "Mon", subheaderText: "27.03.2016")
+
+        let model = AreaBarChartViewModel(mainChart: mainChart, componentCharts: [history, history, history, history, history], horizontalAxisStops: 20, horizontalAxisLabelText: "people billing:", centerText: nil)
```
Comment (karolkozub) [inline GrandCentralBoard/Widgets/Harvest/HarvestWidget.swift]: What happened to the `hotizontalAxisCountLabelText` parameter?

Comment (ochococo) [inline GrandCentralBoard/Widgets/Harvest/HarvestWidget.swift]: Stopped being so HOT 😀 (HOTizontal, you get it?) 🔥

File: GrandCentralBoard/Widgets/Harvest/View/AreaBarChartViewModel.swift [hunk 2]
```diff
@@ -20,21 +20,31 @@ struct AreaBarItemViewModel {
...
     let horizontalAxisStops: Int
     let horizontalAxisLabelText: String
-    let hotizontalAxisCountLabelText: String
+
     let centerText: String?
-    let headerText: String?
-    let subheaderText: String?
 
-    static func emptyViewModel(header header: String? = nil, subheader: String? = nil) -> AreaBarChartViewModel {
+    static func emptyViewModel(header header: String? = nil, subheader: String? = nil, historicalCharts: [AreaBarChartComponentViewModel] = []) -> AreaBarChartViewModel {
```
Comment (karolkozub) [inline GrandCentralBoard/Widgets/Harvest/View/AreaBarChartViewModel.swift]: What if a historical chart is empty? Shouldn't we have a AreaBarChartComponentViewModel.emptyViewModel method as well, and use it inside this method?

**Comment by karolkozub** [comment]
I don't have any more comments. @ochococo

File: GrandCentralBoard/Widgets/Harvest/View/AreaBarChartViewModel.swift (hunk 2 above)
Comment (ochococo) [inline GrandCentralBoard/Widgets/Harvest/View/AreaBarChartViewModel.swift]: You should use empty array `[]` if there is no chart. Do not use empty historical charts. You can specify 0-5.

**Pull request merged**